pkg/runner/types.go

```diff
@@ -213,6 +213,11 @@ var routes = []routev1.Route{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: benchmarkNs,
 			Name:      fmt.Sprintf("%s-passthrough", serverName),
+			Annotations: map[string]string{
+				// Passthrough terminations default balance strategy is source, this strategy is not suitable for
+				// performance testing when concurrency is higher than 1, as all the requests will use the same source IP
+				"haproxy.router.openshift.io/balance": "roundrobin",
```

Comment:
Can we make this configurable via the config.yml and default to roundrobin

Reply:
For comparison with the other termination types is there a reason not to go with 'random'?